app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -199,14 +199,14 @@ class WebViewFragment : Fragment(), NativePlayerHost {
                 val errorMessage = errorResponse.data?.run { bufferedReader().use(Reader::readText) }
                 Timber.e("Received WebView HTTP %d error: %s", errorResponse.statusCode, errorMessage)
 
-                if (request.url == Uri.parse(server.hostname)) onErrorReceived()
+                if (request.url.equals(Uri.parse(view.url))) onErrorReceived()
```

Comment:
Same thing in Kotlin
```suggestion
                if (request.url == Uri.parse(view.url)) onErrorReceived()
```

Reply:
```
/**
* Compares this Uri to another object for equality. Returns true if the
* encoded string representations of this Uri and the given Uri are
* equal. Case counts. Paths are not normalized. If one Uri specifies a
* default port explicitly and the other leaves it implicit, they will not
* be considered equal.
*/
public boolean equals(Object o) {
   if (!(o instanceof Uri)) {
       return false;
   }

   Uri other = (Uri) o;

   return toString().equals(other.toString());
}
```

In this case, wouldn't it be better to skip the Uri parse and compare the strings directly?

`if (request.url.toString() == view.url) onErrorReceived()`

Reply:
I think I deliberately chose to parse it as Uri since the parsing would normalize the URLs, thus I would keep it like that.